doc/reference_cluster.xml

```diff
@@ -145,16 +145,80 @@ GROUP BY cid;
               <xref linkend="ST_ClusterKMeans"/>,
               <xref linkend="ST_ClusterIntersecting"/>,
               <xref linkend="ST_ClusterWithin"/>
+              <xref linkend="ST_ClusterIntersectingWin"/>
           </para>
 	  </refsection>
 
     </refentry>
 
+    <refentry id="ST_ClusterIntersectingWin">
+      <refnamediv>
+        <refname>ST_ClusterIntersectingWin</refname>
+
+        <refpurpose>Window function that returns a cluster id for each input geometry, clustering input geometries into connected sets.</refpurpose>
+      </refnamediv>
+
+      <refsynopsisdiv>
+        <funcsynopsis>
+          <funcprototype>
+            <funcdef>integer <function>ST_ClusterIntersectingWin</function></funcdef>
+            <paramdef><type>geometry winset </type> <parameter>geom</parameter></paramdef>
+          </funcprototype>
+        </funcsynopsis>
+      </refsynopsisdiv>
+
+      <refsection>
+        <title>Description</title>
+
+        <para>ST_ClusterIntersectingWin is a windowing function that builds inter-connecting clusters of geometries that intersect. It is possible to traverse all geometries in a cluster without leaving the cluster. The return value is the cluster number that the geometry argument participates in, or null for null inputs.</para>
+
```

Comment:
This could also be expected to be equivalent to `ST_ClusterDBSCAN(0, 0)`, if not for cases where `ST_Distance(a,b) == 0` but `ST_Intersects(a, b)` is false.